cardano-chain-sync/src/atomic_flow.rs

```diff
@@ -106,12 +106,12 @@ impl<Upstream, Downstream, Cache> AtomicFlow<Upstream, Downstream, Cache> {
         }
     }
 
-    pub async fn run(self)
+    pub async fn run(self, replay_from: Option<Point>)
     where
         Upstream: Stream<Item = ChainUpgrade<MultiEraBlock>> + Unpin + Send,
         Downstream: Sink<(
                 BlockEvents<Either<BabbageTransaction, Transaction>>,
-                TransactionHandle,
+                Option<TransactionHandle>,
             )> + Unpin
             + Send,
         Downstream::Error: Debug,
```

Comment:
This will break downstream projects that depend on it, please make alt method for your use case

Reply:
Reverted all instances of `Option<TransactionHandle>` back to just `TransactionHandle`. But I've kept the 2nd parameter of `run()` function above. Nothing's broken because only `lp-indexer` and `reward-bot` currently use atomic-flow.